src/components/UserSearch/UserSearch.jsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import SearchIcon from "@mui/icons-material/Search";
+import InputAdornment from "@mui/material/InputAdornment";
+import IconButton from "@mui/material/IconButton";
+import { styled } from '@mui/material/styles';
+import styles from './UserSearch.module.css';
+
+// searchbox styling
+const SearchBox = styled(TextField)(() => ({
+  '& fieldset': {
+    borderRadius: '25px',
+    backgroundColor: '#FAFAFC',
+    zIndex: -1,
+  },
+
+  '& .MuiInputBase-input': {
+    width: '250px',
+    color: 'black',
+  },
+
+  '& .MuiOutlinedInput-root': {
+    '&:hover fieldset': {
+      borderColor: '#BFBFBF',
+    },
+    '&.Mui-focused fieldset': {
+      borderColor: '#BFBFBF',
+    },
+  },
+
+}));
+
+export default function UserSearch() {
+
+  const [searchInput, setSearchInput] = React.useState('');
+
+  // log the search input as it's being typed
+  const searchItems = (searchValue) => {
+    setSearchInput(searchValue)
+    console.log(searchInput);
+  }
+
+  // placeholder for the handle search method
+  const handleSearch = () => {
+    console.log('search pressed')
+  };
+
+  return (
+    <Box className={styles.root}
+      component="form"
+      sx={{
+      }}
+      noValidate
+      autoComplete="off"
+    >
+      <SearchBox
+        id="outlined-basic"
+        variant="outlined"
+        placeholder="Type a name"
+        onChange={(e) => searchItems(e.target.value)}
```

Comment:
```suggestion
        onChange={handleSearch}
```
With the above simplifications